apps/back/src/Repository/UserRepository.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\User;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+
+class UserRepository extends ServiceEntityRepository
+{
+
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, User::class);
+    }
+
+    public function add(User $user, bool $flush): void
```

Comment:
Whi did you add `bool $flush` on this method ?

Reply:
in case I need to add more users but only want to flush at the end